### PR TITLE
WEffectselector: make tooltips easier to read

### PR DESCRIPTION
**Pull request by ronso0**
before
image

this PR
image

### CODE REVIEW

File: src/widget/weffectselector.cpp [hunk 1]
```diff
@@ -57,15 +57,13 @@ void WEffectSelector::populate() {
...
-        QString description = tr("%1: %2").arg(pManifest->name(),
-                                               pManifest->description());
+        QString name = tr("%1").arg(pManifest->name());
+        QString description = tr("%1").arg(pManifest->description());
         // The <span/> is a hack to get Qt to treat the string as rich text so
         // it automatically wraps long lines.
-        setItemData(i, QVariant("<span/>" + description), Qt::ToolTipRole);
+        // <b> makes the effect name bold.
+        setItemData(i, QVariant("<span/><b>" + name + "</b><br/>"
+                + description), Qt::ToolTipRole);
```
Comment (Holzhaus) [inline src/widget/weffectselector.cpp:66]: If we start with `<b>` I don't think we need the span anymore. Also, shouldn't we use QStringLiterals here?

File: src/widget/weffectselector.cpp [hunk 2]
```diff
@@ -57,15 +57,13 @@ void WEffectSelector::populate() {
...
         addItem(elidedDisplayName, QVariant(pManifest->id()));
 
-        // NOTE(Be): Using \n instead of : as the separator does not work in
-        // QComboBox item tooltips.
-        // TODO(Be): Check if this is also the case with Qt5.
-        //: %1 = effect name; %2 = effect description
-        QString description = tr("%1: %2").arg(pManifest->name(),
-                                               pManifest->description());
+        QString name = tr("%1").arg(pManifest->name());
+        QString description = tr("%1").arg(pManifest->description());
```
Comment (Holzhaus) [inline src/widget/weffectselector.cpp:61]: Why not this?

```suggestion
        QString name = tr(pManifest->name());
        QString description = tr(pManifest->description());
```

**Review by Holzhaus — CHANGES_REQUESTED** [review]
Thanks!

File: src/widget/weffectselector.cpp (hunk 2 above)
Comment (ronso0) [reply src/widget/weffectselector.cpp:61]: :facepalm: sure..

Comment (ronso0) [reply src/widget/weffectselector.cpp:61]: AFAICT we dont need `tr` here, isn't the translatable string already catched in the effect manifest?
https://github.com/mixxxdj/mixxx/blob/master/src/effects/builtin/bessel8lvmixeqeffect.cpp#L15

**Comment by ronso0** [comment]
Thanks for the quick review.
I addressed the comments and squashed commits to keep this compact.

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you!

**Pull request merged**